tarxiv/lightcurves.py

```diff
@@ -2,14 +2,29 @@
 
 import io
 import requests
+import os
 import logging
-
+import yaml
 import pandas as pd
 
+#########################
+# CONSTANTS CONSTANTS CONSTANTS
+#########################
 _LOG = logging.getLogger(__name__)
 FINKAPIURL = "https://fink-portal.org"
+ATLASAPIURL = "https://star.pst.qub.ac.uk/sne/atlas4/api/"
+ATLASAPI_CONFIG = "/home/stevance/software/st3ph3n/st3ph3n/data/api_config_MINE.yaml"
```

Comment:
Better to put it in a environment variable:

```python
ATLASAPI_CONFIG = os.environ["ATLASAPI_CONFIG"]
```

and then on the machine of the user, we just define the path.